### ISSUE TITLE
`rustup install` is missing from `rustup -h`

### ISSUE THREAD

**Issue by PaulBone** [rust-lang/rustup#1740]
**Problem**
The `rustup install` sub-command is missing from the built-in documentation (`rustup -h`).  I learnt about it here: https://github.com/rust-lang/rustup.rs/blob/master/README.md#toolchain-specification.

Maybe there are other undocumented commands?

Thank you.

**Notes**

Output of `rustup --version`:
rustup 1.16.0 (beab5ac2b 2018-12-06)

Output of `rustup show`:
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

stable-i686-unknown-linux-gnu
stable-x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.32.0 (9fda7c223 2019-01-16)

**Comment by kinnison** [comment]
Hmm, the top level `rustup install` and `rustup uninstall` commands come as part of the `rustup toolchain` command, hence

```rust
                .setting(AppSettings::Hidden) // synonym for 'toolchain install'
```

in `fn cli()` in `src/cli/rustup_mode.rs`.  I think the *correct* fix here is to the `README.md` to use the `rustup toolchain install` form instead.

**Comment by nrc** [comment]
`rustup install` is just an alias for `rustup update` and `update` is documented.

**Comment by PaulBone** [comment]
The fix in #1750 looks good. Thanks.
